lms/djangoapps/courseware/management/commands/dump_course_ids_with_filter.py

```diff
@@ -0,0 +1,38 @@
+"""
+Dump the course_ids available to the lms, excluding courses
+that have ended prior to the given date.
+
+Output is UTF-8 encoded by default.
```

Comment:
The output of this command is a list of course ids, one per line. Is that the best way to make this consumable to the exporter?

Reply:
Ah, I just saw your other PR. I think it makes sense

Reply:
Nit: It would be good to comment on the output format in this docstring, despite how simple the code is.